hack/bazel-build.sh

```diff
@@ -28,6 +28,8 @@ mkdir -p ${CMD_OUT_DIR}/dump
 
 # Build all binaries for amd64
 bazel build \
+    --sandbox_debug \
```

Comment:
This can be added to `.bazelrc` if someone needs it. Do we really want to commit it?

Reply:
Added this by mistake... removed now.